### PR TITLE
Hotfix: TestNet - Substrate 0.9.36 - Preimage pallet

### PR DESCRIPTION
**Pull request by rakanalh**
### Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Dependency upgrade (A change in substrate or any 3rd party crate version)

### Migrations and Hooks

- [ ] This change requires a runtime migration.
- [ ] Modifies `on_initialize`
- [ ] Modifies `on_finalize`

### Checklist

- [ ] Change has been tested locally.
- [ ] Change adds / updates tests.
- [x] Changelog doc updated.

### CODE REVIEW

**Comment by MRamanenkau** [comment]
Why `cere-dev` runtime is not updated?

**Comment by rakanalh** [comment]
> Why `cere-dev` runtime is not updated?

Cere-dev does not have any democracy stuff going on so it's not included in the hotfix but rather will be fixed in the `dev` branch.

**Review by MRamanenkau — APPROVED** [review]

**Review by MRamanenkau — APPROVED** [review]

File: runtime/cere/src/lib.rs [hunk 1]
```diff
@@ -1496,10 +1496,7 @@ parameter_types! {
 	pub DummyPalletAccountId: AccountId = DummyPalletId::get().into_account_truncating();
 }
 /// Runtime migrations
-type Migrations = (
-	pallet_balances::migration::MigrateToTrackInactive<Runtime, DummyPalletAccountId>,
-	pallet_contracts::Migration<Runtime>,
-);
+type Migrations = ();
```
Comment (yahortsaryk) [inline runtime/cere/src/lib.rs:1499]: Probably we will need to provide a migration that moves existing preimages (if any) from the pallet-democracy's internal storage to the external pallet-preimages storage.

Comment (rakanalh) [reply runtime/cere/src/lib.rs:1499]: I checked storage migrations for democracy / preimages & scheduler pallets and there is no such thing. I believe it's going to be time consuming to migrate un-executed preimage call hashes so i think we can skip this part.

**Review by yahortsaryk — APPROVED** [review]

**Pull request merged**